src/_component/connect/connect.scss

```diff
@@ -0,0 +1,121 @@
+.connect__radio:checked + .column .connect__disc{
+  background-color: #d52027;
+}
+
+.box__footer,
+.box__body{
+  padding: 20px;
+}
+
+.box__footer{
+  display: flex;
+  justify-content: center;
+}
+
+@for $i from 1 through 10 {
+  .connect__radio:nth-of-type(#{$i}):checked ~ [data-box="#{$i}"] {
+    display: block;
+  }  
+}
+
+[data-box]{
+display: none;
+}
+
+.box__body{
+  border-top: 2px solid #f5f5f5;
+  border-bottom: 2px solid #f5f5f5;
+}
+
+.box__with-arrow{
+  box-shadow: 0 2px 6px rgba(0, 0, 0, 0.25);
+  border-radius: 6px;
+  background-color: white;
+  position: relative;
+  margin-top: 1rem;
+
+  &::after{
+    content: '';
+    position: absolute;
+    background-color: white;
+    width: 15px;
+    height: 15px;
+    box-shadow: 0 2px 6px rgba(0, 0, 0, 0.25);
+    top: -7px;
+    left: 15px;
+    clip-path: polygon(-6px -6px, 100% 0, 0 100%);
+    transform: rotate(45deg);
+  }
+}
+
+.box__header{
+  padding: 20px;
+  display: flex;
+}
+
+p.box__paragraph{
+  font-size: 14px;
+  line-height: 30px
+}
+
+.box__title{
+  flex-grow: 1;
+  line-height: 40px;
+  font-weight: 500;
+  font-size: 22px;
+  text-transform: capitalize;
+}
+
+.box__close{
+  width: 40px;
+}
+
+.connect__disc{
+  display: block;
+  transition: 250ms;
+  background-color: #1875f0;
+  border-radius: 50%;
+  padding-top: 100%;
+  position: relative;
+  color: white;
+  cursor: pointer;
+
+  [for="connect__label--reset"]{
+    width: 40px;
+    color: white;
+  }
+
+  [class^= "icon__"] {
+    position: absolute;
+    top: 0;
+    left: 0;
+    right: 0;
+    bottom: 0;
+    font-size: 8vw;
+    display: flex;
+    justify-content: center;
+    align-items: center;
+    }
+}
+
+.connect__avatar{
+  padding-top: 100%;
+  background-color: white;
+  position: relative;
+  box-shadow: 0 2px 6px rgba(0, 0, 0, 0.25);
+  border-radius: 8px;
+  transform: rotate(-2.5deg);
+
```

Comment:
One empty line instead of 2, please don't hate me Maria ❤️

Reply:
hahah thanks <3